### DISCUSSION TITLE
Argmax

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1756] [Q&A] [answered]
I have a dataset of stock movements for each day: Stock symbol, opening price, closing price, date.

I want to get, for each stock symbol, the row that has the highest close price (in other words, the row for the date where the stock closed on the highest price for that period.) If there happen to be two dates/rows with the same maximum close price, I still want to get just one row (I don't mind which of the two rows is selected in this case.)

How can I do this with VisiData?

**Comment by cool-RR** [comment]
This is one of the last few pieces I need for my talk.

**Comment by saulpw** [comment, marked as answer]
Have you tried the `keymax` aggregator?

**Reply by cool-RR** [reply]
Thank you, that works.
